perf_test/runHowitzer.sh

```diff
@@ -66,8 +66,9 @@ for TEST in render/*.js; do
   emphasize "Intermediate test results for: ${TEST}"
   cat $RESULT_FILE
 
-#   emphasize "Generating intermediate Markdown for K6 tests..."
-#   python3 -m src.k6.scraper -r results -s summary -p persistent-results -c $CONFIG_FILE -k configs/kingdom.dict
+  emphasize "Generating intermediate Markdown for K6 tests..."
```

Comment:
There's another `python3 -m perf_test.scripts.scraper -r results -s summary -c $CONFIG_FILE` down below that basically does the same command. Is there a difference here? Maybe we only need one?

Reply:
Yeah, so testing it out locally and understanding the code as well, I think we can just remove the invocation in the loop and just running it at the end will suffice
Testing it on the perf cluster to confirm no change in behaviour